### DISCUSSION TITLE
null-ls error when opening files with nvchad

### DISCUSSION THREAD

**Discussion by nstanard** [NvChad/NvChad#1397] [Q&A] [answered]
Hey @siduck, thanks for the previous help. I've gotten further in my setup of NVChad but am running into something I don't understand. I've read all of the [Custom Config docs ](https://nvchad.github.io/config/Walkthrough) like you mentioned before but using the example in the [Format & Lint](https://nvchad.github.io/config/Format%20&%20lint) doc page my instance of NVChad shows an error when opening files. 

Error: `packer.nvim: Error running config for null-ls.nvim: [string "..."]:0: attempt to index a boolean value`

![image](https://user-images.githubusercontent.com/4204806/181591679-7cbfa398-19dd-46ab-bba8-8e49f166417e.png)

Since I'm new to this nvim project I'm not sure if this is an error with null-ls or the NVChad setup. Are you able to point me in the right direction please?

Edit: I should also note, as mentioned before, my custom setup can be found here: https://github.com/nstanard/nvchad-custom/tree/main/custom

I've tried multiple formats for the null-ls.lua file in my `custom/plugins` folder but that doesn't seem to make a difference. Thank you in advance for any help.

**Comment by siduck** [comment, marked as answer]
@nstanard there's no setup function in your null-ls config so remove it

![image](https://user-images.githubusercontent.com/59060246/181784044-44463c50-f65c-4895-ae51-ec27f1a811f2.png)

```lua
require("custom.plugins.null-ls")
```

**Reply by nstanard** [reply]
~Hm, I made this change and ran packer sync but I still get the null-ls error while opening js/ts files.~ 🤔 

Ooph - sorry. I spoke too soon. I restarted nvim after packer sync and that did get rid of this!
